src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/FritekstVedtakbegrunnelser.tsx

```diff
@@ -129,47 +129,49 @@ const FritekstVedtakbegrunnelser: React.FC<IProps> = () => {
+                    <InfoBoks>
                         Fritekst til kulepunkt i brev (valgfri)
                         <StyledHjelpetekst44px
                             type={PopoverOrientering.OverVenstre}
                             innhold={
                                 <div>
-                                    Brev som sendes ut bør være så kortfattede og presise som mulig.{' '}
+                                    <Normaltekst>
+                                        Brev som sendes ut bør være så kortfattede og presise som
+                                        mulig.{' '}
+                                    </Normaltekst>
                                     <Lenke
                                         target="_blank"
                                         rel="noopener noreferrer"
                                         href="https://navno.sharepoint.com/sites/intranett-kommunikasjon/SitePages/Spr%C3%A5k.aspx"
                                     >
                                         Se retningslinjer for klarspråk.
                                     </Lenke>
-                                    <p />
-                                    <b>Eksempler på formulering:</b>
                                     <br />
-                                    <text style={{ fontStyle: 'italic' }}>
+                                    <br />
+                                    <Element>Eksempler på formulering:</Element>
+                                    <Normaltekst style={{ fontStyle: 'italic' }}>
                                         Barnevernet har bekreftet at de overtok omsorgen for barnet
                                         mars 2021
-                                    </text>
-                                    <p />
-                                    <text style={{ fontStyle: 'italic' }}>
+                                    </Normaltekst>
+                                    <br />
+                                    <Normaltekst style={{ fontStyle: 'italic' }}>
                                         Opplysningene fra Folkeregisteret viser at barnet ikke bor
                                         sammen med deg
-                                    </text>
+                                    </Normaltekst>
                                 </div>
                             }
                         />
                         <StyledEtikettInfo mini={true}>
                             Skriv {målform[søkersMålform()]}
                         </StyledEtikettInfo>
-                    </StyledElement>
+                    </InfoBoks>
                     {Object.keys(redigerbarefritekster).map((fritekstId: string) => {
                         return (
-                            <StyledFamilieFritekstFelt>
+                            <StyledFamilieFritekstFelt key={`fritekst-${fritekstId}`}>
                                 <SkjultLegend>{`Kulepunkt ${fritekstId}`}</SkjultLegend>
```

Comment:
Stusser litt over denne. Gir denne id'en mening for SB som bruker skjermoppleser? Er det en id fra 0-3?

Reply:
fritekstId er en teller som starter på 1.